README.md

```diff
@@ -24,6 +24,46 @@ poetry install
 ```
 You can then get a shell with the environment via `poetry shell`.
 
+---
```

Comment:
Even if not related to these changes, when installing as described above, I had to add poetry explicitly to my bash:

`export PATH=$PATH:$HOME/.poetry/bin`

We may consider adding this to the README in case this is a good thing to do.

Reply:
Good idea. Would you mind making a PR for this, please?